statler_site/templates/play-detail-page.html

```diff
@@ -64,6 +66,11 @@
       <div id="details"></div>
       <div style="height: 200px; border-style: solid;">
         Review post box will go here.
+        <form action="{% url 'api:play-reviews' play_id %}" method="post">
+          {% csrf_token %}
```

Comment:
While I agree that form encoding is *a* standard way of sending data, it's not *our* standard way of sending data. To this point, we've written a pure JSON API which could easily be consumed by the site we're writing, a native mobile app, a command line interface, or whatever, so long as it can handle JSON and has our API spec.

I could rant more here, but I don't expect that would be super helpful. Maybe we should hash this out IRL. (After graphics maybe?)

@aseebart, @areinhardt14, and @mnijmeh11, feel free to chime in.